src/test/scala/com/modelfabric/test/FusekiManager.scala

```diff
@@ -36,12 +36,14 @@ object FusekiManager {
     var process: Option[Process] = None
 
     override def run(): Unit = {
+      // JC: another option is to call org.apache.jena.fuseki.cmd.FusekiCmd.main(args), instead of starting a process
```

Comment:
I wanted the test to be as realistic as possible, i.e. using an external process for the test triple store.